github/repos.go

```diff
@@ -55,6 +55,7 @@ type Repository struct {
 	AllowRebaseMerge *bool            `json:"allow_rebase_merge,omitempty"`
 	AllowSquashMerge *bool            `json:"allow_squash_merge,omitempty"`
 	AllowMergeCommit *bool            `json:"allow_merge_commit,omitempty"`
+	Topics           *[]string        `json:"topics,omitempty"`
```

Comment:
There's no need to make it a pointer to a slice, as the slice is a reference type already, so please make this type a `[]string` instead of a `*[]string`.